### PR TITLE
NTC minor refactor

### PR DESCRIPTION
**Pull request by CarlosGamero**
## Changes

Doing some refactor to facilitate addition of new features in the library

## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

File: packages/app/non-translatable-markup/src/methods/isTextTranslatable.spec.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    }
+  })
+
+  it('should return false if text contains only non-translatable content', () => {
+    const testCases = [
+      '\uE101Hello world!\uE102',
+      ' \uE101Hello world!\uE102 \uE101How are you\uE102 ',
+    ]
+
+    for (const testCase of testCases) {
```
Comment (ondrejsevcik) [inline packages/app/non-translatable-markup/src/methods/isTextTranslatable.spec.ts:34]: 🟢 You can use [each](https://vitest.dev/api/#test-each)  for the test cases.

File: packages/app/non-translatable-markup/vite.config.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ export default defineConfig({
   dependencies: Object.keys(packageJson.peerDependencies),
   test: {
     coverage: {
+      exclude: ['src/**/index.ts'],
```
Comment (kibertoad) [inline packages/app/non-translatable-markup/vite.config.ts:14]: do we need all the internal barrel files?

File: packages/app/non-translatable-markup/src/methods/isTextTranslatable.spec.ts (hunk 1 above)
Comment (CarlosGamero) [reply packages/app/non-translatable-markup/src/methods/isTextTranslatable.spec.ts:34]: Absolutely! will change it🙏

File: packages/app/non-translatable-markup/vite.config.ts (hunk 2 above)
Comment (CarlosGamero) [reply packages/app/non-translatable-markup/vite.config.ts:14]: Going to merge as I would need these changes to continue with my task but if you have concerns, please feel free to leave them here and I will address them on follow-up

Comment (kibertoad) [reply packages/app/non-translatable-markup/vite.config.ts:14]: there are two index.ts files - the root one and one inside `methods`. Is second one useful?

Comment (CarlosGamero) [reply packages/app/non-translatable-markup/vite.config.ts:14]: Oh, I see your point, and at the beginning, I thought is was going to be bigger, but looking at the result ... it doesn't make too much sense ... will remove it on follow-up PR

**Pull request merged**